front-end/src/components/RollCallComponents/GroupsComponent.vue

```diff
@@ -99,16 +102,61 @@ export default {
 		}),
 		joinGroup: withErrorHandler(async function (index: number) {
 			http.get(`/calls/join_group/${this.courseId}/${index}`).then((response) => {
+				// If error sweet alert error
+				this.displaySwalGroup(response.data.status);
 				this.getGroups();
 			});
 		}),
+		chooseSwalConfig(message) {
+			let successMessages = ['successJoin', 'successUpdate', 'successEmpty'];
+			let swalConfig = {
+				icon: '',
+				title: '',
+				text: '',
+			};
+			// First do the "icon: and :title" part
+			if (successMessages.includes(message)) {
+				swalConfig.icon = 'success';
+				swalConfig.title = 'Succès !';
+			} else {
+				swalConfig.icon = 'error';
+				swalConfig.title = 'Oups ...';
+			}
+
+			switch (message) {
+				case 'full':
+					swalConfig.text = 'Ce groupe est plein !';
+					break;
+				case 'alreadyInGroup':
+					swalConfig.text = 'Tu es déjà dans ce groupe !';
+					break;
+				case 'successJoin':
+					swalConfig.text = 'Tu as bien rejoint le groupe !';
+					break;
+				case 'successUpdate':
+					swalConfig.text = 'Vous avez bien été répartis dans des groupes !';
+					break;
+				case 'successEmpty':
+					swalConfig.text = 'Les groupes ont bien été vidés !';
+					break;
+				default:
+					swalConfig.text = 'Une erreur est survenue !';
+					break;
+			}
+			return swalConfig;
+		},
+		displaySwalGroup(message) {
+			Swal.fire(this.chooseSwalConfig(message));
+		},
 		createRandomGroups: withErrorHandler(async function () {
 			http.get(`/calls/create_random_groups/${this.courseId}`).then((response) => {
+				this.displaySwalGroup(response.data.status);
 				this.getGroups();
 			});
 		}),
 		emptyGroups: withErrorHandler(async function () {
 			http.get(`/calls/empty_groups/${this.courseId}`).then((response) => {
```

Comment:
Same, if you're using promise.

async = await.
Not async means .then or promise return.